src/applications/facility-locator/components/FacilityTypeDropdown.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { kebabCase } from 'lodash/fp';
+import classNames from 'classnames';
+import Downshift from 'downshift';
+
+import { facilityTypes } from '../config';
+import { keyMap } from '../utils/helpers';
+
+const FACILITY_OPTIONS = ['all', 'health', 'benefits', 'cemetery', 'vet_center'];
+
+const facilityOptionClasses = (item, selected) => classNames(
+  'dropdown-option',
+  { selected },
+  { 'icon-option': item && item !== 'all' },
+  { [`${kebabCase(item)}-icon`]: item && item !== 'all' }
+);
+
+const itemToString = (item) => facilityTypes[item] || 'All Facilities';
+
+const focusDropdown = () => document.querySelector('#facility-dropdown-toggle').focus();
+
+const FacilityTypeDropdown = ({
+  getButtonProps,
+  getItemProps,
+  highlightedIndex,
+  isOpen,
+  selectedItem,
+  toggleMenu
+}) => {
+  const handleKeyDown = (e) => {
+    switch (e.keyCode) {
+      // Allow blurring focus (with TAB) to close dropdown.
+      case keyMap.TAB:
+        if (isOpen) { toggleMenu(); }
+        break;
+
+      // Allow ENTER to toggle state of menu without making a selection.
+      case keyMap.ENTER:
+        toggleMenu();
+        break;
+
+      default: // Do nothing.
+    }
+  };
+
+  const options = FACILITY_OPTIONS.map((item, index) => (
+    <li key={item} {...getItemProps({
+      item,
+      className: facilityOptionClasses(item, index === highlightedIndex),
+      role: 'option',
+      'aria-selected': index === highlightedIndex
+    })}>
+      {itemToString(item)}
+    </li>
+  ));
+
+  /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+  return (
+    <div>
+      <label htmlFor="facility-dropdown-toggle" onClick={focusDropdown}>
+        Select Facility Type
+      </label>
+      <div id="facility-dropdown">
+        <div {...getButtonProps({
```

Comment:
@U-DON You might need to turn on full keyboard access to make Safari work as you'd expect: https://support.apple.com/en-us/ht204434#fullkeyboard

`<button>` should absolutely receive focus by default, but Macs are funny about this behavior. Try that out and see if the button works as you expect. I'll take a closer look at this ASAP, working on a timeboxed item elsewhere this morning.

Reply:
Thanks, @1Copenut. That seemed to be the solution from a user perspective from what I searched earlier, but unsure if we should expect users to know to make that settings change. I feel like it would be more correct to be semantic, and the users that need this functionality would likely have enabled that setting anyway.

I'll assign and add you to the reviewers so you can track it.

Reply:
It does seem to work as expected in Safari with full keyboard access.

Reply:
@U-DON Ideally this is a `<button>` element instead of a div. It has `SPACE` and `ENTER` event listeners, but feels like a good opportunity to opt for semantics.

Reply:
Since we have an ID that matches the label's `htmlFor` declaration, we can remove the `aria-label`. Screen reader users will always hear "Select Facility Type" when the dropdown receives focus that way.

If the `aria-label` is part of the Downshift component and cannot be removed, let's pass it the string "Select Facility Type" to match our visual label.